editor/editor_settings.cpp

```diff
@@ -1483,17 +1483,22 @@ Ref<Shortcut> EditorSettings::get_shortcut(const String &p_name) const {
 	// Use the first item in the action list for the shortcut event, since a shortcut can only have 1 linked event.
 
 	Ref<Shortcut> sc;
+	bool noShortcut = false;
 	const Map<String, List<Ref<InputEvent>>>::Element *builtin_override = builtin_action_overrides.find(p_name);
 	if (builtin_override) {
-		sc.instantiate();
-		sc->set_event(builtin_override->get().front()->get());
-		sc->set_name(InputMap::get_singleton()->get_builtin_display_name(p_name));
+		if (!builtin_override->get().is_empty()) {
+			sc.instantiate();
+			sc->set_event(builtin_override->get().front()->get());
+			sc->set_name(InputMap::get_singleton()->get_builtin_display_name(p_name));
+		} else {
+			noShortcut = true;
+		}
 	}
 
 	// If there was no override, check the default builtins to see if it has an InputEvent for the provided name.
 	if (sc.is_null()) {
 		const OrderedHashMap<String, List<Ref<InputEvent>>>::ConstElement builtin_default = InputMap::get_singleton()->get_builtins().find(p_name);
-		if (builtin_default) {
+		if (builtin_default && noShortcut == false) {
```

Comment:
Two style issues:
- Identifiers should use `snake_case`
- Use `!` operator for booleans instead of `== false`

Reply:
Sorry, i'm fixed.